daq/session_server.py

```diff
@@ -54,16 +53,17 @@ def StartSession(self, request, context):
 class SessionServer:
     """Devices state server"""
 
-    def __init__(self, on_session=None, server_address=None, server_port=None, max_workers=None):
+    def __init__(self, on_session=None, server_address=None, server_port=None, local_ip=None):
         self._return_queues = {}
         self._last_streamed = {}
         self._done_callbacks = {}
         self._lock = threading.Lock()
         self._on_session = on_session
-        self._server = grpc.server(
-            futures.ThreadPoolExecutor(max_workers=max_workers or DEFAULT_MAX_WORKERS))
+        self._local_ip = local_ip
+        self._server = grpc.server(futures.ThreadPoolExecutor())
```

Comment:
Unless we have some specific reason, I'd leave fixing the value out b/c we don't really know what to fix it at, so a default might actually make sense. Otherwise we're surely making an arbitrary decision for something... rather than making no comment on the subject.